### PR TITLE
Fixed recursive flag type

### PR DESCRIPTION
**Pull request by espinosajuanma**
Please refer to the HQ Platform Ticket [#1818](https://hq.slingrs.io/prod/runtime/#views/6164599a65821e3293e7aa5b/readOnly/65cea61c0815720ed2e8303e?backToCollection=true)

### CODE REVIEW

File: src/main/java/io/slingr/service/ftp/beans/Processor.java [hunk 1]
```diff
@@ -99,7 +99,7 @@ public Processor(AppLogs appLogs, Events events, Files files, String name, boole
             }
         }
 
-        this.recursive = Boolean.TRUE.equals(recursive);
+        this.recursive = recursive.equals("enabled");
```
Comment (agreggio-slingr) [inline src/main/java/io/slingr/service/ftp/beans/Processor.java:102]: Use this method
this.recursive = Configuration.parseBooleanValue("enabled");

**Review by agreggio-slingr — COMMENTED** [review]
You can use the following approach, so we avoid type casting and also prevent changing the type in the Processor class.

```java

@ServiceProperty(name = "recursive")
private String recursive;

private Processor processor = null;

@Override
public void serviceStarted() {
    logger.info("Starting FTP service");
    processor = new Processor(appLogs(), events(), files(), properties().getApplicationName(), properties().isLocalDeployment(),
            protocol, host, port, username, password, filePattern, inputFolder, archiveFolder, archiveGrouping,
            Configuration.parseBooleanValue(recursive), outputFolder);
    processor.start();
}
```

**Comment by espinosajuanma** [comment]
@agreggio-slingr done ✨

File: .vscode/settings.json
Comment (agreggio-slingr) [inline .vscode/settings.json:1]: Please add .vscode to .gitignore

Comment (espinosajuanma) [reply .vscode/settings.json:1]: Shame on me. Fixed. I usually use vim, but I needed auto import, so I went to VSCode 💩

**Pull request merged**